hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -938,3 +967,199 @@ def _get_prioritized_host_on_3par(self, host, hosts, hostname):
                 return host, hostname
 
         return host, hostname
+
+    def create_snapshot(self, snapshot):
+        LOG.debug("Create Snapshot\n%s", json.dumps(snapshot, indent=2))
```

Comment:
you can make to LOG.info() since for code tracing it will help if the logging is restrictive.